.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -17,6 +17,16 @@ body:
       description: >
         Shown in the title bar, logs, and long press on the game icon.  If it's outside a game, put 'None.'
 
+ type: input
```

Comment:
This doesn't pass YAML validation, so the issue form would just be broken.  You can check on your fork by viewing the file on github or using yaml lint.  Your fork should even be a good way to preview it.

-[Unknown]

Reply:
hard to me